### ISSUE TITLE
Deployed on Netlify: preview mode is wonky

### ISSUE THREAD

**Issue by joelvarty** [agility/agilitycms-nextjs-starter#21]
- Infinite Redirect on switch to preview
- It doesn't behave quite right after that (reload) either

**Comment by StefanRetief** [comment]
My issue may be related: https://github.com/agility/agility-next/issues/3#issue-947091570

would you be able to confirm you don't have any issues in your console?

**Comment by jamesvidler** [comment]
Hi @StefanRetief we've released an update to fix an issue with infinite redirects while entering Preview Mode when hosted on Netlify. There are still some open issues here though on the Netlify side with how their Next plugin works:

1. Open issue with Netlify where you don't "stay" in preview - this is a known issue on their end they are trying to prioritize for this week
2. The revalidate  tag to revalidate published content is not currently implemented in Netlify. That being said, each CDN node has its own cache right now so its partially implemented but not reliable. That means that content will 'update' periodically depending on what CDN node you are hitting.

So, in summary the preview functionality is still wonky and reliably updating your content without a rebuild is also "flaky" according to Netlify. We are working very closely with their team on this and assisting in testing their updates. This is an issue with all Next.js sites hosted on Netilfy at the moment and not specific to the starter. We'll update this issue as soon as we validate there is a fix.
